remove $'s for easy copying terminal commands

ahmet-umut opened the pull request:
before:
 $ make

after:
make

tohojo reviewed (changes requested):
Looks like you copied the contents of the file so it's included twice now? Please fix that.

The DCO bot is complaining because the commit author is different from the contents of the signed-off-by tag. The commit is authored by "95590170+ahmet-umut@users.noreply.github.com", so I guess you used the github editor to edit the file? No idea how to change the commit author in that case, but I can fix that while merging if you just fix the duplicated text...

ahmet-umut commented:
> Looks like you copied the contents of the file so it's included twice now? Please fix that.
> 
> The DCO bot is complaining because the commit author is different from the contents of the signed-off-by tag. The commit is authored by "[95590170+ahmet-umut@users.noreply.github.com](mailto:95590170+ahmet-umut@users.noreply.github.com)", so I guess you used the github editor to edit the file? No idea how to change the commit author in that case, but I can fix that while merging if you just fix the duplicated text...

Ah, sorry for the duplication. I fixed it (double-checked) and also corrected a link in the table of contents too.
